lms/djangoapps/branding/models.py

```diff
@@ -0,0 +1,23 @@
+"""
+Model for branding
+
+This was implemented initially so that the list of course tiles could be
+stored in this model.
+"""
+
+#from django.contrib import admin
+from django.db import models
+
+from config_models.models import ConfigurationModel
+from xmodule_django.models import CourseKeyField
+
+
+class TileConfiguration(ConfigurationModel):
+    """
+        Stores a list of tiles presented on the front page.
+    """
+    site = models.CharField(max_length=32, default='default', blank=False)
+    course_id = CourseKeyField(max_length=128, db_index=True)
```

Comment:
Is there a constant value somewhere we can reference for the `max_length`? 

We probably have this magic number littered throughout the code base, but it would be nice if we could avoid doing so...